CHANGELOG.md

```diff
@@ -1,12 +1,23 @@
 # Release History
 
 
-## 1.2.0 (TBD)
+## 1.2.0 (2023-03-02)
+
+### New Features
 
-### New Features:
 - Added support for displaying source code as comments in the generated scripts when registering stored procedures. This
 is enabled by default, turn off by specifying `source_code_display=False` at registration.
+- Added a parameter `if_not_exists` when creating udf/udtf/sproc from snowpark python to ignore creating the specified
+function/procedure if it already exists.
+- Accept integers when calling `functions.get` to extract value from array.
+- Added `functions.reverse` in functions to open access to snowflake built-in function
+[reverse](https://docs.snowflake.com/en/sql-reference/functions/reverse).
+- Added parameter `require_scoped_url` in snowpark file api to replace `is_owner_file` is marked for deprecation.
```

Comment:
Suggest `snowflake.snowflake.files.SnowflakeFile.open()` (in Private Preview) instead of "snowpark file api". It's clearer

Reply:
Snowflake, Snowflake, Snowflake. Capitalize it.

Reply:
"api" also needs to be capitalized "API".

Reply:
`snowflake.snowflake.files.SnowflakeFile.open()` this doesnt need capitalization